smaps/src/main/webapp/index.jsp

```diff
@@ -0,0 +1,38 @@
+<!--
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!DOCTYPE html>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<%@ page import="com.google.smaps.Homepage" %>
+<html>
+<head>
+  <link href='//fonts.googleapis.com/css?family=Marmelad' rel='stylesheet' type='text/css'>
+  <title>Smaps Visualizer</title>
+</head>
+<body>
+    <h1>Welcome to the Smaps Visualizer</h1>
+
+  <p><%= Homepage.getInfo() %></p>
```

Comment:
I think the %= and ending % might have been an artifact of when this was "This is%=.....%" so we can probably delete them. (Build/run and make sure it works).

Reply:
I looked into this and it's part of the syntax for JSP (https://www3.ntu.edu.sg/home/ehchua/programming/java/JSPByExample.html). I haven't worked with jsp before but I think it's used instead of just an HTML file because it lets you write java commands within the <%= %>